_tutorials/2016-05-22-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,118 @@
+---
+published: false
+date: "2016-05-22 13:49 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64-quickstart"
+author: Lisa Roach
+excerpt: "Getting started with Cisco's XRv64 Vagrant Instance."
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - XRv64
+---
+### Quick Start Using XRv64 Box image
+Quick guide to get up and running with XRv64 and vagrant/Virtualbox.
+ 
+Install Virtualbox and vagrant tools to your mac:
+
+Source: http://sourabhbajaj.com/mac-setup/Vagrant/README.html
+
+*	Recommend using the Homebrew package manager.
+
+*	Make sure you install version 5.x virtualbox
+
+*	Vagrant latest version is: 1.8.2
+
+*	Website:  http://brew.sh/
+
+        /usr/bin/ruby -e "$(curl -fsSLhttps://raw.githubusercontent.com/Homebrew/install/master/install)"
+        brew cask install virtualbox
+        brew cask install vagrant
+        brew install socat
```

Comment:
I don't see any steps that you use telnet access in. Do you need socat for this tutorial?